instrumentation/opentelemetry-instrumentation-aiohttp-client/setup.py

```diff
@@ -66,6 +68,32 @@
 
 extras_require["test"] = test_deps
 
+
+class JSONMetadataCommand(distutils.cmd.Command):
```

Comment:
Curious if you'd considered putting this class in the instrumentation package for convenience? Wondering if this is something others who want to create instrumentations that are not necessarily in this repo would be able to use.

Reply:
That would add the instrumentation package as a build/setup dependency to all instrumentations. It is a regular dependency anyway so adding a dep is not a concern but I've never used setup dependencies so not sure how well it works or if it has any downsides. 

Instrumentations outside this repo don't really need this as bootstrap command will probably never know about them.

Reply:
Either way, having it in setup.py keeps the doors open if we want to move it to the opentelemetry-instrumentation package later.

Reply:
Fair enough.